ts/features/walletV3/payment/navigation/navigator.tsx

```diff
@@ -20,8 +20,7 @@ const Stack = createStackNavigator<WalletPaymentParamsList>();
 export const WalletPaymentNavigator = () => (
   <Stack.Navigator
     initialRouteName={WalletPaymentRoutes.WALLET_PAYMENT_MAIN}
-    headerMode={"screen"}
-    screenOptions={{ gestureEnabled: isGestureEnabled }}
+    screenOptions={{ gestureEnabled: isGestureEnabled, headerShown: true }}
```

Comment:
@mastro993  check here
```suggestion
    screenOptions={{ gestureEnabled: isGestureEnabled, headerMode: true }}
```